software_tests

```diff
@@ -1,13 +1,10 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
`set -e` to make the script actually fail when there is an error, `set -x` for logging, `set -u` to fail on missing environment variables, `set -o pipefail` as a best practice.